lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -1328,7 +1328,7 @@ template <typename T> bool Equivalent(T l, T r) {
 
 /// Specialization for GetTypeInfo().
 template <> bool Equivalent<uint32_t>(uint32_t l, uint32_t r) {
-  if (l != r) {
+  if (l < r) {
```

Comment:
This works, but it also weakens the GetTypeInfo comparison quite a bit. A trick you could do is wrap the result in Optional, then it uses the GetBitSize equivalent function which I think is already accepting >=

Reply:
(line 1520)

Reply:
I had to do a few tweaks to reuse it. See 1cb9b37.